### PR TITLE
systemc 2.3.1 (new formula)

### PR DESCRIPTION
**Pull request by theknix**
The C++ modeling language known as "SystemC".

Includes the SystemC library and some examples.

### CODE REVIEW

File: Library/Formula/systemc.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  url "http://accellera.org/images/downloads/standards/systemc/systemc-2.3.1.tgz"
+  sha256 "7ce0f68fd4759e746a9808936b54e62d498f5b583e83fc47758ca86917b4f800"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
```
Comment (bfontaine) [inline Library/Formula/systemc.rb]: Please add a `test do` block.

**Comment by theknix** [comment]
I added a simple test with https://github.com/Homebrew/homebrew/commit/a6b69f887add3113065dcb78a1d11c714bf35a3f

File: Library/Formula/systemc.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--with-arch-suffix=",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  # compile and run a simple systemc program
```
Comment (DomT4) [inline Library/Formula/systemc.rb]: Don't need this comment :)

File: Library/Formula/systemc.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
+class Systemc < Formula
+  desc "The core SystemC language and examples"
```
Comment (DomT4) [inline Library/Formula/systemc.rb]: Can probably leave out the `The` here.

**Comment by theknix** [comment]
@DomT4 fixed.

**Comment by bfontaine** [comment]
Thank you for your contribution to Homebrew @theknix! :tada:

**Pull request closed without merging**